modules/core/test/src/smithy4s/http/StringAndBlobSpec.scala

```diff
@@ -75,6 +75,27 @@ class StringAndBlobSpec() extends munit.FunSuite {
     expect.same(Right(input), roundTripped)
     expect.same(HttpMediaType("text/csv"), mediaType)
   }
+  test("String Enum  ") {
+    val input = StringEnumBody(StringEnum.INTERESTING)
```

Comment:
hmm INTERESTING code :D

Reply:
Lol , I have had one too many foo and bar